### PR TITLE
Add delete message to SQS table

### PR DESCRIPTION
**Pull request by hunterwerlla**
- Add delete message action to SQS messages table
- Like the S3 object delete, it's very difficult to test with unit tests, so we will test it with UI tests
## Screenshots
Kapture 2020-09-02 at 09 10 04

## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -10,3 +10,4 @@ Build/
 venv/
 guitest.log
 stale_outputs_checked
+**/wrapper/
```
Comment (abrooksv) [inline .gitignore:13]: Why was this added?

Comment (hunterwerlla) [reply .gitignore:13]: Because we keep getting wrappers in `jetbrains-ultimate` after running tests :( . It won't exclude already included files, the only thing it does it make it require an additional flag to update the gradle wrapper.

Comment (abrooksv) [reply .gitignore:13]: Where it is putting wrappers? I have not seen that lcoally

Comment (hunterwerlla) [reply .gitignore:13]: `jetbrains-ulatimate/wrapper`

Comment (hunterwerlla) [reply .gitignore]: Reverted `.gitignore` change

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/actions/DeleteMessageAction.kt [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        }
+        launch {
+            try {
+                val requestEntries = messages.mapIndexed { index, item ->
+                    // ID must be unique per request. Since we don't need to use it, just use index (other fields can be too long)
+                    DeleteMessageBatchRequestEntry.builder().id(index.toString()).receiptHandle(item.receiptHandle()).build()
+                }
+                val response = client.deleteMessageBatch { it.queueUrl(queueUrl).entries(requestEntries) }
+                // Remove all of the selected rows
+                messages.forEach { _ ->
```
Comment (rli) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/actions/DeleteMessageAction.kt:51]: should we block polling while deleting messages?

File: resources/resources/software/aws/toolkits/resources/localized_messages.properties [hunk 3]
```diff
@@ -736,6 +736,10 @@ sqs.create.standard.label=Standard
 sqs.create.validation.empty.queue.name=Queue name must be specified.
 sqs.create.validation.long.queue.name=Queue name must not exceed {0} characters in length.
 sqs.create.validation.queue.name.invalid=Queue names can only contain alphanumeric characters, hyphens (-), and underscores (_).
+sqs.delete.message.action=Delete {0, choice, 1#Message|2#{0,number} Messages}
```
Comment (kiiadi) [inline resources/resources/software/aws/toolkits/resources/localized_messages.properties:739]: nit: I'd got with lowercase "m" - "Delete 1 message" / "Delete 3 messages"

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/actions/DeleteMessageAction.kt (hunk 2 above)
Comment (hunterwerlla) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/actions/DeleteMessageAction.kt:51]: Probably, will change

File: resources/resources/software/aws/toolkits/resources/localized_messages.properties (hunk 3 above)
Comment (hunterwerlla) [reply resources/resources/software/aws/toolkits/resources/localized_messages.properties:739]: Sure

**Review by rli — APPROVED** [review]
